webpack.config.js

```diff
@@ -30,7 +30,21 @@ const config = {
           {
             loader: 'babel-loader',
             options: {
-              presets: [['@babel/env', { modules: false }], '@babel/react'],
+              presets: [
+                ['@babel/preset-env',
+                  {
+                    modules: false,
+                    targets: {
+                      chrome: '88',
+                      ie: '11',
+                      edge: '88',
+                      firefox: '79',
+                      safari: '13.1.3',
+                    },
+                  }
+                ],
```

Comment:
- `@babel/env` was mistakenly added, but not used; replaced with `@babel/preset-env`